File: TransCoder_tokenized_test_set_functions/cpp/REARRANGE_ARRAY_ORDER_SMALLEST_LARGEST_2ND_SMALLEST_2ND_LARGEST.cpp
void rearrangeArray ( int arr [ ], int n ) {
  sort ( arr, arr + n );
  int tempArr [ n ];
  int ArrIndex = 0;
  for ( int i = 0, j = n - 1;
  i <= n / 2 || j > n / 2;
  i ++, j -- ) {
    tempArr [ ArrIndex ] = arr [ i ];
    ArrIndex ++;
    tempArr [ ArrIndex ] = arr [ j ];
    ArrIndex ++;
  }
  for ( int i = 0;
  i < n;
  i ++ ) arr [ i ] = tempArr [ i ];
}

File: TransCoder_tokenized_test_set_functions/python/REARRANGE_ARRAY_ORDER_SMALLEST_LARGEST_2ND_SMALLEST_2ND_LARGEST.py
def rearrangeArray ( arr, n ) :
    arr . sort ( )
    tempArr = [ 0 ] * ( n + 1 )
    ArrIndex = 0
    i = 0
    j = n - 1
    while ( i <= n //2orj>n//2):
        tempArr[ArrIndex]=arr[i]
        ArrIndex=ArrIndex+1
        tempArr[ArrIndex]=arr[j]
        ArrIndex=ArrIndex+1
        i=i+1
        j=j-1
    foriinrange(0,n):
        arr[i]=tempArr[i]
    
